test/Microsoft.AspNet.Mvc.Razor.Test/TagHelperContentWrapperTextWriterTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Text;
+using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.Razor
+{
+    public class TagHelperContentWrapperTextWriterTest
```

Comment:
Should be able to remove these tests since the class will be private and will be an implementation detail of `EndTagHelperWritingScope` which is tested above.